lib/install/elm.rb

```diff
@@ -12,9 +15,9 @@
           "#{Webpacker::Configuration.entry_path}/hello_elm.js"
 
 puts "Installing all elm dependencies"
-run "yarn add elm elm-webpack-loader"
-run "yarn add --dev elm-hot-loader"
-run "yarn run elm package install -- --yes"
+run "#{node_bundler} elm elm-webpack-loader"
+run "#{node_bundler} --dev elm-hot-loader"
```

Comment:
`npm -D` won't work, as you'll need (at least) to specify the npm command, `install`.